src/pages/index.tsx

```diff
@@ -2,21 +2,17 @@ import { ReactElement } from 'react'
 import BaseNewLayout from 'layouts/base-new'
 import ProductCardGrid from 'components/product-card-grid'
 import HomepageHero from 'components/homepage-hero'
-import Footer from 'components/footer'
 import s from './index.module.css'
 
 function Homepage(): ReactElement {
   return (
-    <>
-      <div className={s.root}>
-        <HomepageHero className={s.hero} />
-        <h2 className={s.cardGridHeading}>Explore Product Documentation</h2>
-        <ProductCardGrid className={s.cardGrid} />
-      </div>
-      <Footer className={s.footer} />
-    </>
+    <div className={s.root}>
+      <HomepageHero className={s.hero} />
+      <h2 className={s.cardGridHeading}>Explore Product Documentation</h2>
+      <ProductCardGrid className={s.cardGrid} />
+    </div>
   )
 }
 
-Homepage.layout = BaseNewLayout
+Homepage.layout = (p) => <BaseNewLayout {...p} showFooter />
```

Comment:
Have you already weighed the pros/cons of the approach you've taken compared to exporting a `getStaticProps` function that sets `layoutProps` like this?

export const getStaticProps: GetStaticProps = async () => {
  return {
    props: {
      layoutProps: {
        showFooter
      },
    },
  }
}

Reply:
Right on, we can address this over there no problem!